python/paddle/fluid/tests/unittests/test_power.py

```diff
@@ -0,0 +1,167 @@
+        print("-----test2----")
+        dims = (np.random.randint(200, 300), )
+        x = (np.random.rand(*dims) * 10).astype(np.float64)
+        y = int(np.random.rand() * 10)
+        res = _run_power('dynamic', x, y)
+        self.assertTrue(np.allclose(res, np.power(x, y)))
+        res = _run_power('static', x, y)
+        self.assertTrue(np.allclose(res, np.power(x, y)))
+
+        print("-----test3----")
+        x = (np.random.rand(*dims) * 10).astype(np.int64)
+        y = int(np.random.rand() * 10)
+        res = _run_power('dynamic', x, y)
+        self.assertTrue(np.allclose(res, np.power(x, y)))
+        res = _run_power('static', x, y)
+        self.assertTrue(np.allclose(res, np.power(x, y)))
+        print("-----test4----")
+        # test 1-d float tensor ** 1-d float tensor
+        dims = (np.random.randint(200, 300), )
+        x = (np.random.rand(*dims) * 10).astype(np.float64)
+        y = (np.random.rand(*dims) * 10).astype(np.float64)
+        res = _run_power('dynamic', x, y)
+        self.assertTrue(np.allclose(res, np.power(x, y)))
+        res = _run_power('static', x, y)
+        self.assertTrue(np.allclose(res, np.power(x, y)))
+        print("-----test5----")
+        # test 1-d float tensor ** 1-d int tensor
+        dims = (np.random.randint(200, 300), )
+        x = (np.random.rand(*dims) * 10).astype(np.float64)
+        y = (np.random.rand(*dims) * 10).astype(np.int64)
+        res = _run_power('dynamic', x, y)
+        self.assertTrue(np.allclose(res, np.power(x, y)))
+        res = _run_power('static', x, y)
+        self.assertTrue(np.allclose(res, np.power(x, y)))
+        print("-----test6----")
+        # test 1-d int tensor ** 1-d float tensor
+        dims = (np.random.randint(200, 300), )
+        x = (np.random.rand(*dims) * 10).astype(np.int64)
+        y = (np.random.rand(*dims) * 10).astype(np.float64)
+        # print(np.power(x, y))
+        res = _run_power('dynamic', x, y)
+        self.assertTrue(np.allclose(res, np.power(x, y)))
+        res = _run_power('static', x, y)
+        self.assertTrue(np.allclose(res, np.power(x, y)))
+        print("-----test7----")
+        # test 1-d int tensor ** 1-d int tensor
+        dims = (np.random.randint(200, 300), )
+        x = (np.random.rand(*dims) * 10).astype(np.int64)
+        y = (np.random.rand(*dims) * 10).astype(np.int64)
+        res = _run_power('dynamic', x, y)
+        self.assertTrue(np.allclose(res, np.power(x, y)))
+        res = _run_power('static', x, y)
+        self.assertTrue(np.allclose(res, np.power(x, y)))
+        print("-----test8----")
+        # test broadcast
+        dims = (np.random.randint(1, 10), np.random.randint(5, 10),
+                np.random.randint(5, 10))
+        x = (np.random.rand(*dims) * 10).astype(np.float64)
+        y = (np.random.rand(dims[-1]) * 10).astype(np.float64)
+        res = _run_power('dynamic', x, y)
+        self.assertTrue(np.allclose(res, np.power(x, y)))
```

Comment:
You can also add some tests for returning types, not only values to prevent the float32 I commented below.